src/components/_common/card/Card.module.scss

```diff
@@ -1,5 +1,28 @@
 @use 'src/common' as common;
 
+@mixin highlight {
+    > :global(.situation) {
+        background-color: var(--a-orange-200);
+    }
+    > :global(.product) {
+        background-color: var(--a-green-200);
+    }
+    > :global(.tool) {
+        background-color: var(--a-deepblue-200);
+    }
+    > :global(.themed-article) {
+        background-color: var(--a-limegreen-300);
+    }
+    > :global(.guide) {
+        background-color: var(--a-lightblue-200);
+    }
+    //TODO: Find a better way? (can't specify classes in Card Size modules here or vice versa)
+    * {
+        text-decoration: none;
+        color: var(--a-gray-900) !important;
+    }
```

Comment:
Kanskje denne kan løses uten `!important` om den ikke puttes i mixin?

Reply:
Hmm, er litt usikker på hvorfor dette er løst slik og når denne egentlig slår inn. Dette er bare kode jeg har flyttet på, så det er kanskje litt out of scope for denne oppgaven uansett.

Reply:
Ok. Om det ikke er vanskelig å finne ut av en løsning som unngår `!important`, synes jeg det er fint å ta sammen med denne oppgaven. Ellers ingen krise å ta senere.